Automattic/woocommerce-payments
# Display a deposits paused notice when the account has a negative available balance

Fixes #5730 

#### Changes proposed in this Pull Request

This PR introduces a notice shown to merchants when their available balance is negative and may lead to paused deposits.

<img width="708" alt="Screenshot 2023-04-18 at 1 28 57 pm" src="https://user-images.githubusercontent.com/8490476/232663679-3c1c98bc-3e17-423f-9fc3-05e3d36aa56d.png">

#### Testing instructions





1. Turn on the new deposit UI: `update_option( '_wcpay_feature_simplify_deposits_ui', 1 );`
2. Purchase any product using a test card that bypasses the pending period (`4000000000000077`)
3. Refund the newly created order. 
4. Go to the Payments > Overview screen.
5. Note that the negative balance notice is shown (see screenshot above). You may need to wait a minute for the transaction to flow through.
6. Make another purchase. 
7. Note that the negative balance notice is no longer displayed. 

-------------------


**Post merge**
